BoardTest.py

```diff
@@ -0,0 +1,50 @@
+import board
+import unittest
+
+
+class BoardTest(unittest.TestCase):
+    def test_delta(self):
+        r = 'A1'
```

Comment:
a) In an assertEqual, the constant (expected) value should always come first.

b) I think we are standardizing on lower case...

c) Unnecessarily verbose.  You can achieve the same by saying:

self.assertEqual('b2', board.delta('a1', 1, 1)